sdk/python/tests/unit/infra/online_store/test_dynamodb_online_store.py

```diff
@@ -175,6 +177,34 @@ def test_online_read(repo_config, n_samples):
     assert [item[1] for item in returned_items] == list(features)
 
 
+@mock_dynamodb2
+def test_online_read_unknown_entity(repo_config):
+    """Test DynamoDBOnlineStore online_read method."""
+    n_samples = 2
+    _create_test_table(PROJECT, f"{TABLE_NAME}_{n_samples}", REGION)
+    data = _create_n_customer_test_samples(n=n_samples)
+    _insert_data_test_table(data, PROJECT, f"{TABLE_NAME}_{n_samples}", REGION)
+
+    entity_keys, features, *rest = zip(*data)
+    # Append a nonsensical value
+    entity_keys = list(entity_keys)
+    entity_keys.append(
+        EntityKeyProto(
+            join_keys=["customer"], entity_values=[ValueProto(string_val="12359")]
+        )
+    )
+    features = list(features)
+    features.append(None)
+    dynamodb_store = DynamoDBOnlineStore()
+    returned_items = dynamodb_store.online_read(
+        config=repo_config,
+        table=MockFeatureView(name=f"{TABLE_NAME}_{n_samples}"),
+        entity_keys=entity_keys,
+    )
+    assert len(returned_items) == len(entity_keys)
+    assert [item[1] for item in returned_items] == list(features)
```

Comment:
Wanna assert that the nonsense key has none values?

Reply:
Done